src/main/java/frc/utils/brakestate/BrakeStateManager.java

```diff
@@ -0,0 +1,28 @@
+package frc.utils.brakestate;
+
+
+import java.util.ArrayList;
+
+public class BrakeStateManager {
+
+	private static final ArrayList<Runnable> brakeRunnables = new ArrayList<>();
+	private static final ArrayList<Runnable> coastRunnables = new ArrayList<>();
+
+	public static void add(Runnable brake, Runnable coast) {
+		brakeRunnables.add(brake);
+		coastRunnables.add(coast);
+	}
```

Comment:
create funcs for only one

Reply:
do fucn that gets coast consumer and another one for brake consumer

Reply:
But why two funcs? In what situation would you want only coast or brake? If you add only a brake when you enable it will just stay as brake until you turn off the robot